HTMessaging/Classes/Connection.swift

```diff
@@ -9,8 +9,17 @@
 import Foundation
 import Firebase
 
+// MARK: Initialization
+
 public final class Connection {
-    func withFirebaseFile(with fileName: String, ofType: Filetype, completion: @escaping(_ status: FIRResponse?) -> Void) {
+    
+    /**
+    Performs a connection to Firebase with the File's name and file type.
+    @param fileName The filename of te Google-services.
+    @param ofType The type of the Google-services file.
+    @param completion The handler block to execute. status is the value of the response(Ex. Successful) .
+    */
+    static func withFirebaseFile(with fileName: String, ofType: Filetype, completion: @escaping(_ status: FIRResponse?) -> Void) {
```

Comment:
one last change, sorry, can you add a param name to ofType? maybe "type"?

it would be: ofType type: Filetype